_topic_maps/_topic_map.yml

```diff
@@ -1134,6 +1134,8 @@ Topics:
     File: cert-manager-operator-issuer-acme
   - Name: Configuring certificates with an issuer
     File: cert-manager-creating-certificate
+  - Name: Securing routes with the {cert-manager-operator}
```

Comment:
It did not render well in the preview I think. It shows `{cert-manager-operator}` as is. 
```
Security and compliance / cert-manager Operator for Red Hat OpenShift / Securing routes with the {cert-manager-operator}
```